daemon/pool.go

```diff
@@ -32,6 +36,31 @@ var (
 	errDriverStopped = errors.NewKind("driver stopped")
 )
 
+var (
+	policyDefaultWindow    = envIntOr("BBLFSHD_POLICY_WINDOW", 30)
+	policyDefaultMin       = envIntOr("BBLFSHD_POLICY_MIN", 1)
+	policyDefaultScale     = envIntOr("BBLFSHD_POLICY_SCALE", 1)
```

Comment:
Is this a multiplier or a cap?

Reply:
Added a `_INC` suffix.

Reply:
I'm not sure what `_INC` means in this context—is that an additive increment for the AIMD policy?